From 5fd06b565a27def8b0f7e8d8a265901a1f3721e2 Mon Sep 17 00:00:00 2001
From: Pascal Paillet <p.paillet@foss.st.com>
Date: Wed, 2 Oct 2024 16:14:37 +0200
Subject: [PATCH 1253/1695] drivers: regulator: change tree trace level

Change regulator print tree trace level so that
it can bee seen when it is requested by xtest.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 47e2897ca9cbd439ab47dae8a5255b2ffe1f2e5f drivers: regulator: change tree trace level]
Change-Id: Ie5c4e8309d17beaf53f8fac1d04be47bb2f45033
Signed-off-by: Pascal Paillet <p.paillet@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/408813
Tested-by: Pascal PAILLET <p.paillet@st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
Reviewed-by: Pascal PAILLET <p.paillet@st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Etienne CARRIERE <etienne.carriere@foss.st.com>
Domain-Review: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
---
 core/drivers/regulator/regulator.c | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/core/drivers/regulator/regulator.c b/core/drivers/regulator/regulator.c
index 4d2dd9203..bc5f12d28 100644
--- a/core/drivers/regulator/regulator.c
+++ b/core/drivers/regulator/regulator.c
@@ -339,7 +339,8 @@ static TEE_Result regulator_core_cleanup(void)
 		}
 	}
 
-	regulator_print_tree();
+	if (TRACE_LEVEL >= TRACE_DEBUG)
+		regulator_print_tree();
 
 	return TEE_SUCCESS;
 }
@@ -476,7 +477,7 @@ out:
 	if (!msg)
 		snprintf(msg_end - 4, 4, "...");
 
-	DMSG("%s", msg_buf);
+	IMSG("%s", msg_buf);
 }
 
 static void print_tree(void)
@@ -514,11 +515,10 @@ static void print_tree(void)
 
 void regulator_print_tree(void)
 {
-	if (IS_ENABLED(CFG_DRIVERS_REGULATOR_PRINT_TREE) &&
-	    TRACE_LEVEL >= TRACE_DEBUG) {
-		DMSG("Regulator tree summary");
+	if (IS_ENABLED(CFG_DRIVERS_REGULATOR_PRINT_TREE)) {
+		IMSG("Regulator tree summary");
 		if (SLIST_EMPTY(&regulator_device_list))
-			DMSG("-- No registered regulator");
+			IMSG("-- No registered regulator");
 		else
 			print_tree();
 	}
-- 
2.47.2

